python/ray/serve/_private/deployment_state.py

```diff
@@ -290,7 +290,7 @@ def start(self, deployment_info: DeploymentInfo, version: DeploymentVersion):
             deployment_info.deployment_config.is_cross_language
         )
 
-        logger.debug(
+        logger.info(
```

Comment:
nit: For consistency to show replica operation, also add corresponding info in the grace_stop() func

Reply:
done

Reply:
done